src/virtualenv/seed/embed/wheels/acquire.py

```diff
@@ -139,6 +153,8 @@ def download_wheel(packages, for_py_version, to_folder):
     # pip has no interface in python - must be a new sub-process
     process = Popen(cmd, env=pip_wheel_env_run("{}{}".format(*sys.version_info[0:2])), stdout=subprocess.PIPE)
     process.communicate()
+    if process.returncode != 0:
+        raise RuntimeError("failed to download wheels")
```

Comment:
these three lines could be `check_output(...)`

Reply:
it cannot, as we patch Popen to support unicode paths on python 2 🤷‍♂ so we need to use the custom popen 👍

Reply:
oh lol